src/components/VaccinationDateChoiceForm.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import Calendar from "react-calendar";
```

Comment:
Hmmm
Chyba dobrze by było to ujednolicić z tym co jest w EnterSchedulePage, ale to bardzo low prio

Reply:
Musisz powiedzieć chyba o co chodzi dokładniej, w releasie będę pracował nad ujednoliceniem. Na razie zostawiam.

Reply:
Żeby widget kalendarza był ten sam
tutaj to jest react-calendar, tam co innego
Chyba, że źle pamiętam, to wtedy nvm